assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -59,41 +59,43 @@ function printView(url) {
+								<input type="checkbox" checked="checked" name="notGrade" value="true"/> $tlang.getString("defaultGrade.notGrade")&nbsp;
 							</label>
 							<label class="textPanelFooter">
-								<input type="checkbox" checked="checked" name="notSubmit" value="true"/>$tlang.getString("defaultGrade.notSubmit")<br/>
+								<input type="checkbox" checked="checked" name="notSubmit" value="true"/> $tlang.getString("defaultGrade.notSubmit")<br/>
 							</label>
 						</div>
-					</div>
-					#if ($!gradeType != 3)
-						<select id="defaultGrade_1" name="defaultGrade" size="1" >
-							<option value="">$tlang.getString("non.submission.grade.select")</option>
-							#if ($!gradeType == 1)
-								<option value="gen.nograd" #if($!defaultGrade && $!defaultGrade.equals("gen.nograd"))selected="selected"#end>$tlang.getString("gen.nograd")</option>
-							#elseif ($!gradeType == 2)
-								#foreach ($i in $letterGradeOptions)
-									<option value="$i" #if($!defaultGrade && $!defaultGrade.equals($i))selected="selected"#end>$i</option>
-								#end
-							#elseif ($!gradeType == 4)
-								#foreach ($i in ["Pass", "Fail"])
-									<option value="$i" #if($!defaultGrade && $!defaultGrade.equals($i))selected="selected"#end>#if ($i=="Pass") $tlang.getString("pass") #elseif ($i=="Fail") $tlang.getString("fail") #end</option>
-								#end
-							#elseif ($!gradeType == 5)
-								<option value="Checked" #if($!defaultGrade && $!defaultGrade.equals("Checked"))selected="selected"#end>$tlang.getString("gen.checked")</option>
+						<div class="flex-wrap">
+							#if ($!gradeType != 3)
+								<select id="defaultGrade_1" name="defaultGrade" size="1" >
+									<option value="">$tlang.getString("non.submission.grade.select")</option>
+									#if ($!gradeType == 1)
+										<option value="gen.nograd" #if($!defaultGrade && $!defaultGrade.equals("gen.nograd"))selected="selected"#end>$tlang.getString("gen.nograd")</option>
+									#elseif ($!gradeType == 2)
+										#foreach ($i in $letterGradeOptions)
+											<option value="$i" #if($!defaultGrade && $!defaultGrade.equals($i))selected="selected"#end>$i</option>
+										#end
+									#elseif ($!gradeType == 4)
+										#foreach ($i in ["Pass", "Fail"])
+											<option value="$i" #if($!defaultGrade && $!defaultGrade.equals($i))selected="selected"#end>#if ($i=="Pass") $tlang.getString("pass") #elseif ($i=="Fail") $tlang.getString("fail") #end</option>
+										#end
+									#elseif ($!gradeType == 5)
+										<option value="Checked" #if($!defaultGrade && $!defaultGrade.equals("Checked"))selected="selected"#end>$tlang.getString("gen.checked")</option>
+									#end
+								</select>
+							#else
+							## instructor needs to type in default points in point-based-grading
+							<input type="number" id="defaultGrade_2" name="defaultGrade" value="$!defaultGrade" min="0" step="0.01" placeholder="$tlang.getString('gen.gra')" />
```

Comment:
This CAN'T be type="number" because we need grade like this 9,65 on Spanish. 
Have you tested on Spanish language?

Reply:
I believe this relies on the browser and I think the number type is the right choice here because a number is expected, positive or greater than 0 with a split of two decimals.

What I don't know if this input allows to introduce letters or P/NP in other submission types, any clue @V100X ?

Reply:
I tested the HTML spec and Chrome in Spanish uses the comma:
![image](https://user-images.githubusercontent.com/8653754/161254575-4c4f9b4a-b818-4074-9083-b04be8d7c860.png)

Reply:
The other type of submission grading use a selector to choose between the different options, so there shouldn't be problem.

Reply:
Thanks @mpellicer